webapp-django/pytest.ini

```diff
@@ -0,0 +1,6 @@
+# pytest configuration for running webapp tests
+[pytest]
+# -rsxX        - show skipped, failed, and passed tests
+# --tb=native  - print native traceback
+addopts = -rsxX --tb=native
+DJANGO_SETTINGS_MODULE = crashstats.settings.test
```

Comment:
We run the tests for the webapp from within this directory, so pytest finds this file first and these become the settings used.

Reply:
We should remove the bit in `settings/__init__.py` that apply test settings since this now handles setting those.

Reply:
Good catch!